compiler/rustc_mir_transform/src/simplify_branches.rs

```diff
@@ -2,20 +2,19 @@ use crate::MirPass;
 use rustc_middle::mir::*;
 use rustc_middle::ty::TyCtxt;
 
-/// A pass that replaces a branch with a goto when its condition is known.
-pub struct SimplifyConstCondition {
-    label: String,
-}
-
-impl SimplifyConstCondition {
-    pub fn new(label: &str) -> Self {
-        SimplifyConstCondition { label: format!("SimplifyConstCondition-{}", label) }
-    }
+pub enum SimplifyConstConditionPassName {
```

Comment:
(nit) no need to block the PR on this, but if you touch this code again in the future could you rename this to just `SimplifyConstCondition`? That would make it consistent with all the other ones

Reply:
Thanks for this, finally got around to this fix.
https://github.com/rust-lang/rust/pull/110657